metalift/synthesize_cvc5.py

```diff
@@ -104,10 +104,18 @@ def toExpr(
     expr_uni = {"not": Not}
     if isinstance(ast, list):
         if ast[0] in expr_bi.keys():
-            return expr_bi[ast[0]](
-                toExpr(ast[1], funName, returnType, varType, letVars),
-                toExpr(ast[2], funName, returnType, varType, letVars),
-            )
+            if len(ast) == 3:
+                return expr_bi[ast[0]](
+                    toExpr(ast[1], funName, returnType, varType, letVars),
+                    toExpr(ast[2], funName, returnType, varType, letVars),
+                )
+            elif len(ast) == 2 and ast[0] == "-":
+                return expr_bi[ast[0]](
+                    toExpr("0", funName, returnType, varType, letVars),  # type: ignore
```

Comment:
Ooh actually instead of this we can simplify the code and get rid of `type: ignore` if we just replace this with `IntLit(0)` directly.

Reply:
`toExpr` expects the first argument to be a `List[Any]` but I believe `IntLit(0)` is actually an `Expr`

Reply:
^ Misunderstood, fixed.